statsmodels/discrete/discrete_model.py

```diff
@@ -3195,8 +3193,6 @@ def fit(self, start_params=None, method='bfgs', maxiter=35,
             full_output=1, disp=1, callback=None, use_transparams = False,
             cov_type='nonrobust', cov_kwds=None, use_t=None, **kwargs):
         """
-        Parameters
-        ----------
```

Comment:
why not?

Reply:
Because of `fit.__doc__ = DiscreteModel.fit.__doc__ + fit.__doc__` below produces two `Parameters --------`. Appender/Substituter would be much clearer here.